### PR TITLE
Added grade spelling eval, improved requirements

### PR DESCRIPTION
**Pull request by JayBaileyCS**
GPT-J doesn't answer the prompts correctly - it always responds with ???? or _____ or similar. I wonder if we need to use logits or some other method to get this information via prompting? 

Figured I'd push this anyway since it contains a list of (word, answer) pairs by grade, which is useful for other methods like probing.

Also added some extra needed requirements to the requirements.txt file

### CODE REVIEW

**Comment by JayBaileyCS** [comment]
Adding more stuff and need to test it, will open a different PR.
